README.md

```diff
@@ -17,48 +17,62 @@ Conda instructions are provided in more detail, but you may also use `pip` and `
 **Note:** Java 11 is required. Instructions for installing Java 11 for Ubuntu or macOS are provided in the [Install with Conda](#install-with-conda) section.
 
 ### Install with pip
+
 To use `pip` to install TorchServe and the model archiver:
 
-```
-pip install torch torchtext torchvision sentencepiece
+``` bash
+pip install torch torchtext torchvision sentencepiece psutil future
 pip install torchserve torch-model-archiver
 ```
 
 ### Install with Conda
-_Ubuntu_
+
+#### Ubuntu
 
 1. Install Java 11
+
     ```bash
     sudo apt-get install openjdk-11-jdk
     ```
+
 1. Install Conda (https://docs.conda.io/projects/conda/en/latest/user-guide/install/linux.html)
 1. Create an environment and install torchserve and torch-model-archiver
     For CPU
+
     ```bash
-    conda create --name torchserve torchserve torch-model-archiver pytorch torchtext torchvision -c pytorch -c powerai
+    conda create --name torchserve torchserve torch-model-archiver psutil future pytorch torchtext torchvision -c pytorch -c powerai
     ```
+
     For GPU
+
     ```bash
-    conda create --name torchserve torchserve torch-model-archiver pytorch torchtext torchvision cudatoolkit=10.1 -c pytorch -c powerai
+    conda create --name torchserve torchserve torch-model-archiver psutil future pytorch torchtext torchvision cudatoolkit=10.1 -c pytorch -c powerai
     ```
+
 1. Activate the environment
+
     ```bash
     source activate torchserve
     ```
 
-_macOS_
+#### macOS
 
 1. Install Java 11
+
     ```bash
     brew tap AdoptOpenJDK/openjdk
     brew cask install adoptopenjdk11
     ```
+
 1. Install Conda (https://docs.conda.io/projects/conda/en/latest/user-guide/install/macos.html)
 1. Create an environment and install torchserve and torch-model-archiver
+
     ```bash
-    conda create --name torchserve torchserve torch-model-archiver pytorch torchtext torchvision -c pytorch -c powerai
+    conda create --name torchserve torchserve torch-model-archiver psutil future pytorch torchtext torchvision -c pytorch -c powerai
```

Comment:
```suggestion
    conda create --name torchserve torchserve torch-model-archiver psutil future pytorch sentencepiece torchtext torchvision -c pytorch -c powerai
```

Reply:
@aaronmarkham Ack, but I think this is necessary in the conda command for Mac as well.

Reply:
^^^^ Disregard...hadn't seen your second change.